src/dhcpv6.c

```diff
@@ -553,6 +553,15 @@ int init_dhcpv6(const char *ifname)
 	na_mode = config_dhcp->ia_na_mode;
 	pd_mode = config_dhcp->ia_pd_mode;
```

Comment:
Maybe we could rewrite this as:
```
if (config_dhcp->allow_slaac_only) {
		na_mode = IA_MODE_NONE;
		pd_mode = IA_MODE_NONE;
} else {
		na_mode = config_dhcp->ia_na_mode;
		pd_mode = config_dhcp->ia_pd_mode;
}
```
This way it's more clear that we're overriding the config.